idioms-json-2.0-custom/ioc-indicator.json

```diff
@@ -10,6 +10,7 @@
             "modified": "2020-02-24T16:07:27.201Z",
             "name": "Zeus",
             "type": "malware",
+            "x_elevator_other_names": "twexts,sdra64,ntos",
```

Comment:
Would it be better to represent this as a list of strings? instead of the comma-separated string...

Reply:
I think this was fixed in the next PR